manager/integration/tests/test_cloning.py

```diff
@@ -140,14 +327,90 @@ def test_cloning_interrupted():  # NOQA
             requests:
               storage: 10Gi
         ```
-    5. Specify the `cloned-pvc` in a cloned pod yaml and deploy the cloned pod
-    6. Wait for the `CloneStatus.State` in `cloned-pvc` to be `initiated`
-    7. Kill all replicas process of the `source-pvc`
-    8. Wait for the `CloneStatus.State` in `cloned-pvc` to be `failed`
-    9. In 2-min retry loop, verify cloned pod cannot start
-    10. Clean up cloned pod and `clone-pvc`
-    11. Redeploy `cloned-pvc` and clone pod
-    12. In 3-min retry loop, verify cloned pod become running
-    13. `cloned-pvc` has the same data as `source-pvc`
-    14. Cleanup the test
+    5. Wait for the `CloneStatus.State` in `cloned-pvc` to be `initiated`
+    6. Kill all replicas process of the `source-pvc`
+    7. Wait for the `CloneStatus.State` in `cloned-pvc` to be `failed`
+    8. Clean up `clone-pvc`
+    9. Redeploy `cloned-pvc` and clone pod
+    10. In 3-min retry loop, verify cloned pod become running
+    11. `cloned-pvc` has the same data as `source-pvc`
+    12. In 2-min retry loop, verify the volume of the `clone-pvc`
+        eventually becomes healthy.
     """
+    # Step-1
+    source_pvc_name = 'source-pvc' + generate_random_suffix()
+    pvc['metadata']['name'] = source_pvc_name
+    pvc['spec']['storageClassName'] = 'longhorn'
+    core_api.create_namespaced_persistent_volume_claim(
+        body=pvc, namespace='default')
+    wait_for_pvc_phase(core_api, source_pvc_name, "Bound")
+
+    # Step-2
+    pod_name = 'source-pod' + generate_random_suffix()
+    pod['metadata']['name'] = pod_name
+    pod['spec']['volumes'] = [create_pvc_spec(source_pvc_name)]
+    create_and_wait_pod(core_api, pod)
+
+    # Step-3
+    write_pod_volume_random_data(core_api, pod_name,
+                                 '/data/test', DATA_SIZE_IN_MB_3)
```

Comment:
Can we modify the test description to specify that we only write 500MB of data?

Reply:
Fixed, thanks